src/frontend/src/common/api/index.tsx

```diff
@@ -13,6 +13,7 @@ export enum API {
   GET_FASTA_URL = "/api/sequences/getfastaurl",
   USHER_TREE_OPTIONS = "/api/usher/tree_options",
   SAMPLES_VALIDATE_IDS = "/api/samples/validate-ids",
+  LOCATIONS = "/v2/locations/",
```

Comment:
I noticed this is the only endpoint with a trailing forward slash. Is that intentional?

Reply:
Yes, our FastAPI setup is particular about having a trailing slash in the url.